addons/mail/static/src/owl/components/composer_text_input/composer_text_input.xml

```diff
@@ -1,9 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
+    <t t-name="owl.UserStatus" owl="1">
+        <span t-att-class="partnerID ? 'o_mention_updatable_im_status' : ''" t-att-data-partner-id="partnerID">
+            <i t-if="status == 'online'" class="o_mail_user_status o_user_online fa fa-circle" title="Online" role="img" aria-label="User is online"/>
+            <i t-if="status == 'away'" class="o_mail_user_status o_user_idle fa fa-circle" title="Idle" role="img" aria-label="User is idle"/>
+            <i t-if="status == 'offline'" class="o_mail_user_status fa fa-circle-o" title="Offline" role="img" aria-label="User is offline"/>
+        </span>
+    </t>
+
     <t t-name="mail.component.ComposerTextInput" owl="1">
         <div class="o_ComposerTextInput">
-            <textarea class="o_ComposerTextInput_textarea" t-esc="storeProps.composer.textInputContent" placeholder="Write something..." t-on-keydown="_onKeydownTextarea" t-on-input="_onInputTextarea" t-ref="textarea" />
+            <div class="dropup o_ComposerTextInput_mention_dropdown">
+                <div class="dropdown-menu" t-ref="mention_dropdown" style="max-width: 1398.48px;" t-att-class="{ 'show': state.isMentionsOpen }">
+                    <t t-if="state.mentionSuggestions[0]" >
+                        <t t-foreach="state.mentionSuggestions[0]" t-as="mention" t-key="mention.id">
+                            <div role="separator" t-if="mention.divider" class="dropdown-divider"/>
```

Comment:
t-if t-else on a separate tag

Reply:
I won't change it right now as I'll need it to split mentions based on current object and all partners. BUT I'll put the t-if t-else in a <t> then